### PR TITLE
Feature/333213 guidance page part2

### PR DESCRIPTION
**Pull request by aldeez988**

### CODE REVIEW

File: app/views/endemics/guidance/apply-guidance-for-farmers.njk [hunk 1]
```diff
@@ -197,11 +197,11 @@
             <p>You must agree with the vet what they’ll focus on. Ask them to read the <a href="#">[vet guidance].</a>
             </p>
             <h2 id="after-the-review">What happens on a review or follow-up</h2>
-            <p>Read the guidance about <a href="../farmer/guidance-visit.html">what happens on a review or follow-up.</a>
+            <p>Read the guidance about <a href="https://fcp-ahwr-prototype.herokuapp.com/v26/farmer/guidance-visit">what happens on a review or follow-up.</a>
```
Comment (govTechSatish) [inline app/views/endemics/guidance/apply-guidance-for-farmers.njk:200]: Why those are prototype links? just use #, if they are not confirmed yet

File: app/views/endemics/guidance/apply-guidance-for-farmers.njk [hunk 2]
```diff
@@ -197,11 +197,11 @@
...
             </p>
             <h2 id="on-the-farm">Apply for a review or follow-up</h2>
             <p>
-              <a href="../apply/start.html">Apply for a review or follow-up.</a>
+              <a href="https://fcp-ahwr-prototype.herokuapp.com/v26/apply/start">Apply for a review or follow-up.</a>
```
Comment (govTechSatish) [inline app/views/endemics/guidance/apply-guidance-for-farmers.njk:204]: Why those are prototype links? just use #, if they are not confirmed yet

File: app/views/endemics/guidance/apply-guidance-for-farmers.njk [hunk 3]
```diff
@@ -212,7 +212,7 @@
               <p>Rural Payments Agency<br>PO Box 352<br>Worksop<br>S80 9FG<br></p>
               <p>Email: <a href="mailto:ruralpayments@defra.gov.uk" class="govuk-link">{{ruralPaymentsAgency.email}}</a><br></p>
                <p>Telephone: {{ruralPaymentsAgency.telephone}}<br> Monday to Friday, 8:30am to 5pm, except public holidays<br>
-                <a href="https://www.gov.uk/call-charges" target='_blank' class="govuk-link">Find out about call charges (opens in a new tab)</a>
```
Comment (govTechSatish) [inline app/views/endemics/guidance/apply-guidance-for-farmers.njk:215]: This seems right to me. Target should be blank

File: app/views/endemics/guidance/apply-guidance-for-farmers.njk [hunk 4]
```diff
@@ -242,7 +242,7 @@
           <nav role="navigation" class="gem-c-related-navigation__nav-section govuk-!-margin-bottom-3" aria-labelledby="related-nav-related_items-da209313" data-module="gem-toggle" data-gem-toggle-module-started="true">
             <ul class="gem-c-contents-list__list govuk-list govuk-list--spaced govuk-!-margin-top-3">
               <li>
-                <a class="govuk-link"  href="https://fcp-ahwr-prototype.herokuapp.com/v26/farmer/guidance-visit">What happens on a livestock health and welfare review or endemic disease follow-up</a>
+                <a class="govuk-link"  href=" https://fcp-ahwr-prototype.herokuapp.com/v25/farmer/guidance-visit">What happens on a livestock health and welfare review or endemic disease follow-up</a>
```
Comment (govTechSatish) [inline app/views/endemics/guidance/apply-guidance-for-farmers.njk:245]: Why those are prototype links? just use #, if they are not confirmed yet

**Review by govTechSatish — CHANGES_REQUESTED** [review]

File: app/views/endemics/guidance/apply-guidance-for-farmers.njk (hunk 1 above)
Comment (aldeez988) [reply app/views/endemics/guidance/apply-guidance-for-farmers.njk]: > Why those are prototype links? just use #, if they are not confirmed yet

done

File: app/views/endemics/guidance/apply-guidance-for-farmers.njk (hunk 2 above)
Comment (aldeez988) [reply app/views/endemics/guidance/apply-guidance-for-farmers.njk]: > Why those are prototype links? just use #, if they are not confirmed yet

done

File: app/views/endemics/guidance/apply-guidance-for-farmers.njk (hunk 3 above)
Comment (aldeez988) [reply app/views/endemics/guidance/apply-guidance-for-farmers.njk]: > This seems right to me. Target should be blank

done

File: app/views/endemics/guidance/apply-guidance-for-farmers.njk (hunk 4 above)
Comment (aldeez988) [reply app/views/endemics/guidance/apply-guidance-for-farmers.njk]: > Why those are prototype links? just use #, if they are not confirmed yet

done

Comment (aldeez988) [reply app/views/endemics/guidance/apply-guidance-for-farmers.njk]: > Why those are prototype links? just use #, if they are not confirmed yet

done

**Pull request merged**